installer/functions.sh

```diff
@@ -125,645 +121,586 @@ service_cmd="$(which_cmd service)"
 rcservice_cmd="$(which_cmd rc-service)"
 systemctl_cmd="$(which_cmd systemctl)"
 service() {
-    local cmd="${1}" action="${2}"
-
-    if [ ! -z "${systemctl_cmd}" ]
-    then
-        run "${systemctl_cmd}" "${action}" "${cmd}"
-        return $?
-    elif [ ! -z "${service_cmd}" ]
-    then
-        run "${service_cmd}" "${cmd}" "${action}"
-        return $?
-    elif [ ! -z "${rcservice_cmd}" ]
-    then
-        run "${rcservice_cmd}" "${cmd}" "${action}"
-        return $?
-    fi
-    return 1
+	local cmd="${1}" action="${2}"
+
+	if [ ! -z "${systemctl_cmd}" ]; then
+		run "${systemctl_cmd}" "${action}" "${cmd}"
+		return $?
+	elif [ ! -z "${service_cmd}" ]; then
+		run "${service_cmd}" "${cmd}" "${action}"
+		return $?
+	elif [ ! -z "${rcservice_cmd}" ]; then
+		run "${rcservice_cmd}" "${cmd}" "${action}"
+		return $?
+	fi
+	return 1
 }
 
 # -----------------------------------------------------------------------------
 # portable pidof
 
 pidof_cmd="$(which_cmd pidof)"
 pidof() {
-    if [ ! -z "${pidof_cmd}" ]
-    then
-        ${pidof_cmd} "${@}"
-        return $?
-    else
-        ps -acxo pid,comm |\
-            sed "s/^ *//g" |\
-            grep netdata |\
-            cut -d ' ' -f 1
-        return $?
-    fi
+	if [ ! -z "${pidof_cmd}" ]; then
+		${pidof_cmd} "${@}"
+		return $?
+	else
+		ps -acxo pid,comm |
+			sed "s/^ *//g" |
+			grep netdata |
```

Comment:
backslashes at end of line removed.
is this ok?

Reply:
yes it is